apache/docker-entrypoint.sh

```diff
@@ -147,11 +147,16 @@ EOPHP
 
 $stderr = fopen('php://stderr', 'w');
 
-list($host, $port) = explode(':', $argv[1], 2);
+list($host, $port_or_socket) = explode(':', $argv[1], 2);
+if ( 0 !== strpos( $port_or_socket, '/' ) ) {
+  $port = (int)$port_or_socket;
+} else {
+  $socket = $port_or_socket;
+}
```

Comment:
I think we probably ought to use either `is_numeric` or `file_exists` here instead of forcing it to start with `/`:

```php
list($host, $socket) = explode(':', $argv[1], 2);
$port = 0;
if (is_numeric($socket)) {
	$port = (int) $socket;
	$socket = null;
}
```

(be sure to use tabs for indentation like the surrounding code)

I'm also happy to take over the PR from here if you'd rather! :+1: